collection-ncsync.nomad

```diff
@@ -0,0 +1,201 @@
+job "collection-ncsync" {
```

Comment:
Once https://github.com/liquidinvestigations/node/pull/17 is merged, we should use collection templates for the synced collection.